module.exports = {
  name: 'Dmitry Penzar',
  title: 'Bioinformatician, ML enthusiast, teacher',
  facts: {
    'Residence': 'Moscow',
    'Github': `<a href="https://github.com/dmitrypenzar1996/">
                  <span class="print">github</span>
                  <span class="screen">dmitrypenzar1996</span>
                </a>`,
    'Email': '<a href="mailto:dmitrypenzar1996@gmail.com">dmitrypenzar1996@gmail.com</a>'
  },
  skills: [
    ['Python', '+++++'],
    ['R', '+++++'],
    ['C/C++', '+++'],
    ['Golang', '++' ],
    ['Statistics', '+++++'],
    ['Classical ML', '+++++'],
    ['Deep learning', '++++'],
    ['Algorithms', '+++++'],
    ['Molecular biology', '+++'],
    ['Bioinformatics', '+++++'],
    ['English\n(speaking)', '+++'],
    ['English\n(technical literature)', '+++++'],
  ],
  plays: [
    {
      title: 'Department of Computational Systems Biology, VIGG RAN',
      written: 'June 2017 - Current',
      badges: ['ML', 'biology', 'healthcare'],
      contents: `
      Advisor: Dr. Ivan Kulakovskiy. 
      Classical ML and Deep Neural Networks application for the regulatory mutations effect  
`
    },
    {
      title: 'Belozersky Institute of Physico-Chemical Biology, Department of Mathematical Methods in Biology',
      written: 'September 2016 - June 2017',
      badges: ['C/C++', 'phylogeny', 'software development'],
      contents: ` 
      Advisor: S.A. Spirin
      We developed a novel program, named PQ, for reconstructing protein and nucleic acid phylogenies following a new character-based principle. 
`
    },
    {
      title: 'Internship in Netherlands, LUMC, Molecular Epidemiology, Department of Medical Statistics and Bioinformatics',
      written: 'July 2016',
      badges: ['R', 'NGS', 'software development'],
      contents: `
      Advisor: Dr. Szymon Kielbasa
      R-Shiny based NGS data quality control and outliers detection toolkit Zeeuw
	    `
    },
    {
      title: 'Technosphere, Mail.Ru Group',
      written: 'October 2016 - December 2017',
      badges: ['R', 'NGS', 'software development'],
      contents: `
      Intense two-year program by a leading Russian IT company. Within the program, I participated in differeent ML Kaggle compeetitions, aquired a huge experience in ML and software development, deepened knowledge of C/C++. I quited from the third semester of the program due to job offer from Insilico Medicine company 
   ` 
    },
    {
      title: 'Developer in startup curated by AIAR Labs Inc, department of machine learning',
      written: 'August 2017 - October 2017',
      badges: ['ML', 'DNN', 'software development'],
      contents: `
      NDA(:( )
   ` 
    },
    {
      title: 'Developer in InSilico Medicine, department of machine learning',
      written: 'November 2017 - August 2018',
      badges: ['ML', 'DNN', 'drug design', 'software development'],
      contents: `
      I've been working on generating new drugs using deep neural networks
   `} 
  ],
  experience: [
    {
      header: 'Additional courses',
      skills: ['ecomomics', 'linguistics', 'mathematics'],
      contents: `
I've attend a huge variety of additional courses, including:
1. Categorial grammar
2. Neuroscience and neuromarketing
3. Macromolecular modeling
4. Pattern recognition
5. Mathematical models in biology
6. Mathematical linguistics
`
    },
    {
      header: 'Teacher',
      skills: ['teaching', 'communication', 'team leader'],
      contents: `
      I've been trying to take part in teaching since I was a third-year student. Firstly, I've started as an teacher assistant,
      then - as one of the course authors, now I'm continue to teach different courses, including those led by me.
      Short list of courses I'm taking part in:
      1. Algorithms 
      2. Machine learning in biology (the course led by me)
      3. Introduction to Python
      4. Advanced Python
      5. Deep learning in science
      `
    }
  ],
	publication:[
		{
			header: "Insights gained from a comprehensive all-against-all transcription factor binding motif benchmarking study",
		theme: ["epigenomics", "visualization", "regulatory regions"],
		contents: `
		`},
		{
		header: "The species Severe acute respiratory syndrome-related coronavirus: classifying 2019-nCoV and naming it SARS-CoV-2",
		theme: ["virology", "R", "software development"],
		contents: `
		`},
		{
		header: "H3K4me3, H3K9ac, H3K27ac, H3K27me3 and H3K9me3 Histone Tags Suggest Distinct Regulatory Evolution of Open and Condensed Chromatin Landmarks",
		theme: ["epigenomics"],
		contents: `
		`},
		{
			header: "Integration of Multiple Epigenomic Marks Improves Prediction of Variant Impact in Saturation Mutagenesis Reporter Assayheader: ",
		theme: ["epigenomics", "machine learning", "regulatory regions"],
		contents: `
		`},
		{
			header: "Retroelementâ€”Linked Transcription Factor Binding Patterns Point to Quickly Developing Molecular Pathways in Human Evolution<Paste>",
		theme: ["epigenomics", "visualization", "regulatory regions"],
		contents: `
		`},
		{
			header: "What Do Neighbors Tell About You: The Local Context of Cis-Regulatory Modules Complicates Prediction of Regulatory Variants",
		theme: ["machine learning", "data leakage", "regulatory regions"],
		contents: `
		`},
		{
			header: "PQ, a new program for phylogeny reconstruction",
		theme: ["phylogeny"],
		contents: `
		`},
		{
			header: "Profiling of Human Molecular Pathways Affected by Retrotransposons at the Level of Regulation by Transcription Factor Proteins",
		theme: ["epigenomics", "visualization", "regulatory regions"],
		contents: `
		`},
 ]
};
